dom-creation/main.ts

```diff
@@ -0,0 +1,118 @@
+    imageUrl: 'images/charmander.png',
+  },
+  {
+    number: '007',
+    name: 'Squirtle',
+    description:
+      'When it retracts its long neck into its shell, it squirts out water with vigorous force.',
+    imageUrl: 'images/squirtle.png',
+  },
+  {
+    number: '002',
+    name: 'Ivysaur',
+    description:
+      'When the bulb on its back grows large, it appears to lose the ability to stand on its hind legs.',
+    imageUrl: 'images/ivysaur.png',
+  },
+  {
+    number: '005',
+    name: 'Charmeleon',
+    description:
+      'It has a barbaric nature. In battle, it whips its fiery tail around and slashes away with sharp claws.',
+    imageUrl: 'images/charmeleon.png',
+  },
+  {
+    number: '008',
+    name: 'Wartortle',
+    description:
+      'It is recognized as a symbol of longevity. If its shell has algae on it, that Wartortle is very old.',
+    imageUrl: 'images/wartortle.png',
+  },
+  {
+    number: '003',
+    name: 'Venusaur',
+    description:
+      'Its plant blooms when it is absorbing solar energy. It stays on the move to seek sunlight.',
+    imageUrl: 'images/venusaur.png',
+  },
+  {
+    number: '006',
+    name: 'Charizard',
+    description:
+      'It spits fire that is hot enough to melt boulders. It may cause forest fires by blowing flames.',
+    imageUrl: 'images/charizard.png',
+  },
+  {
+    number: '009',
+    name: 'Blastoise',
+    description:
+      'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png',
+  },
+];
+
+function renderPokemon(pokemon: Pokemon): HTMLDivElement {
+  const columnDiv = document.createElement('div');
+  columnDiv.className = 'column-third';
+
+  const cardDiv = document.createElement('div');
+  cardDiv.className = 'pokemon-card';
+  columnDiv.appendChild(cardDiv);
+
+  const imgElement = document.createElement('img');
+  imgElement.src = pokemon.imageUrl;
+  cardDiv.appendChild(imgElement);
+
+  const textDiv = document.createElement('div');
+  textDiv.className = 'pokemon-card-text';
+  cardDiv.appendChild(textDiv);
+
+  const nameElement = document.createElement('h2');
+  nameElement.textContent = pokemon.name;
+  textDiv.appendChild(nameElement);
+
+  const numberElement = document.createElement('h3');
+  numberElement.textContent = `#${pokemon.number}`;
+  textDiv.appendChild(numberElement);
+
+  const descriptionElement = document.createElement('p');
+  descriptionElement.textContent = pokemon.description;
+  textDiv.appendChild(descriptionElement);
+
+  return columnDiv;
+}
+
+const $row = document.querySelector('.row');
+if (!$row) throw new Error('The $row query failed');
+
+// const pokemonElement = renderPokemon(pokedex[8]);
+// console.log(pokemonElement);
+
+// for (const pokemon of pokedex) {
+//   $row.appendChild(renderPokemon(pokemon));
+// }
+
+for (let i = 0; i < pokedex.length; i++) {
```

Comment:
While this shouldn't show up in your JS when you compile, it's generally good to remove commented out code that isn't being used unless they are specific comments on the code possibly for functionality